SPEECH.md

```diff
@@ -6,6 +6,215 @@ We assume you have already set up a bot and have Web Chat running on a page.
 
 > Sample code in this article is optimized for modern browsers. You may need to use a [transpiler](https://en.wikipedia.org/wiki/Source-to-source_compiler) (e.g. [Babel](https://babeljs.io/)) to target a broader range of browsers.
 
+## Support matrix
+
+<table>
+  <thead>
+    <tr>
+      <th></th>
+      <th></th>
+      <th colspan="2">Chrome/Edge<br />and Firefox<br />on desktop</th>
+      <th colspan="2">Chrome<br />on Android</th>
+      <th colspan="2">Safari<br />on macOS/iOS</th>
+      <th colspan="2"><a href="https://developer.android.com/reference/android/webkit/WebView">Web View<br />on Android</a></th>
+      <th colspan="2"><a href="https://developer.apple.com/documentation/webkit/wkwebview">Web View<br />on iOS</a></th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>STT</td>
+      <td>Basic recognition</td>
+      <th>✔</th><td>4.6</td>
+      <th>✔</th><td>4.6</td>
+      <th>✔</th><td>4.6</td>
+      <th>✔</th><td>4.6</td>
+      <th>❌</th><td><a href="#notes"><sup>*1</sup></a></td>
```

Comment:
Link to individual numbers? the anchors don't appear to work either, just refreshes the page. I don't know if this is an issue from being a PR